lib/browser/index.js

```diff
@@ -162,5 +164,24 @@ Object.defineProperties(Realm, {
     },
 });
 
-// The session ID refers to the Realm constructor object in the RPC server.
-Realm[keys.id] = rpc.createSession();
+for (let i = 0, len = debugHosts.length; i < len; i++) {
+    try {
+        // The session ID refers to the Realm constructor object in the RPC server.
+        Realm[keys.id] = rpc.createSession(debugHosts[i] + ':' + debugPort);
+        break;
+    } catch (e) {
+        // Only throw exception after all hosts have been tried.
+        if (i < len - 1) {
+            continue;
+        }
+
+        // Log the original exception for debugging purposes.
+        console.error(e);
```

Comment:
That's the point since it's almost always going to try multiple IP addresses, so we'd just be spamming the console if we logged every error.